algorithm/cgal/Unfold.h

```diff
@@ -73,15 +79,15 @@ static int Unfold(
                   computeInverseSegmentTransform(s1, s2, Vector(0, 0, 1));
               geometry->segments(edge).emplace_back(s1, s2);
               geometry->setTransform(edge, t.inverse());
-              emit_tag(edge, std::string("unfold:edge"));
+              tags.emplace_back(edge, "unfold:edge");
```

Comment:
What is the current status of unfold?

Reply:
Works, but haven't really had much use of it so far.